src/nvim/vterm/state.c

```diff
@@ -645,7 +645,7 @@ static int on_escape(const char *bytes, size_t len, void *user)
 
     {
       int setnum = bytes[0] - 0x28;
-      VTermEncoding *newenc = vterm_lookup_encoding(ENC_SINGLE_94, bytes[1]);
+      VTermEncoding *newenc = vterm_lookup_encoding();
```

Comment:
I think instead we should delete this entire `case` block with a comment indicating only UTF-8 is supported. Leaving the code here provides the (wrong) impression that we are actually handling other character sets.

Reply:
See my [other comment](https://github.com/neovim/neovim/pull/31934/files#r1910927726), I think we should keep the DEC drawing encoding (removing the others is probably fine), in which case this should stay as-is.